packages/plugin-server-session/session.js

```diff
@@ -60,12 +61,12 @@ const sendSessionSummary = client => sessionCounts => {
   req(handleRes)
 
   function handleRes (err) {
-    if (!err) return client._logger.info('Session delivered')
+    if (!err) return client._logger.debug(`${sessionCounts.length} sessions reported`)
```

Comment:
This should be reworded somewhat. I was confused at first to send a dozen requests but then the console read "1 sessions reported", since it is counting the groups rather than the sessions.